docs/plugins/composable_resources.md

```diff
@@ -276,6 +276,29 @@ acPostProcForDataCopyReceived(*leaf_resource) {
 }
 ```
 
+Detached mode was added as a configuration option for UnixFileSystem resources in release 4.3.1.  When detached mode is enabled, any server may serve up requests for the resource as long as all servers share a common mounted vault.
+
+To enable detached mode, the host_mode setting in the context string should be set to "detached" as in the following example:
+
+```
+irods@hostname:~/ $ iadmin mkresc detached_resc3 unixfilesystem hostname.example.org:/common/mount/point "host_mode=detached"
+```
+
+If an administrator wants to restrict the number of hosts that may serve the request, a host_list parameter with a comma separated list of hosts may be added to the context string.
+
+```
+irods@hostname:~/ $ iadmin mkresc detached_resc3 unixfilesystem hostname.example.org:/common/mount/point "host_mode=detached;host_list=host2.example.org,host3.example.org"
+```
+
+The host assigned to the host field for the resource is assumed to be able to serve up any request, so in the above case either hostname, host2, or host3 may serve up the request.
+
+The following are some rules around using detached mode:
+
+1. If host_list does not exist but host_mode=detached, all hosts are assumed to be able to service requests for the resource.
+2. If host_list does exist, only the hosts in either this list or the host in the resource host field will be able to service requests for the resource.
+3. If a host list exists and the current host is not in the list, redirects will go to the host listed in the resource host field as usual. If that host is down the request will fail even if other hosts in host_list are up.
+4. The path to the resource vault path must be identical on every host that services requests for the resource. This vault path must be in a mount to a common filesystem.
+5. Registrations outside the vault are not allowed as we can't enforce that the path being registered is shared by all hosts that service requests for the resource.
```

Comment:
Consider splitting this statement into multiple statements.

Reply:
Done